README.md

```diff
@@ -631,7 +631,8 @@ which is passed on to the user (removes the administrative fields)
 Returns a normalized task which represents a tuple in the space
 * `tube:take()` - sets the task state to 'in progress' and returns the task.
 If there are no 'ready' tasks in the queue, returns nil.
-* `tube:delete(task_id)` - deletes a task from the queue
+* `tube:delete(task_id)` - deletes a task from the queue (move the task to
+   the 'executed' state)
```

Comment:
Technically there is no such state:

```
$ cat queue/abstract/state.lua 
-- task states
return {
    READY   = 'r',
    TAKEN   = 't',
    DONE    = '-',
    BURIED  = '!',
    DELAYED = '~',
}
```

Reply:
In fact it is not exhaustive list of actions that a driver should do at `tube:delete()` call. Say it should also notify wathers with `on_task_change()`. But it can be deduced from existing drivers code. Writing comprehensive driver API documentation is out of scope of this issue.

Reply:
I would say that the method should return the original task with a state changed to 'done'. 'Move the task to <...> state' looks unclear for me, because the task is to be deleted: current wording arises the question 'where it should be changed?'.

Reply:
Update.

Reply:
Now the change is the following:

```diff
-* `tube:delete(task_id)` - deletes a task from the queue
+* `tube:delete(task_id)` - return the original task with a state changed to 'done'
```

Sorry for the nitpicking, but I think both actions should be mentioned: delete a task from a storage, which the queue driver use (usually a space), return the deleted task tuple with DONE (`'-'`).

Reply:
Updated.

Reply:
Thanks!